.gitignore

```diff
@@ -4,6 +4,8 @@ __pycache__/
 *$py.class
 
 # C extensions
+numcodecs/**/*.c
+numcodecs/**/*.h
```

Comment:
Had forgotten that. Thanks!

Reply:
No worries. Did as well. Plus being able to filter the headers meant needing to drop the one that was here.